tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs

```diff
@@ -70,11 +118,11 @@ public async Task CanBuildAndRun_Angular6App_WithoutCompressedNodeModules(string
                 });
         }
 
-        [Fact]
-        public async Task CanBuildAndRun_Angular6App_With_NodeModule_Dir_Exists_InRoot_WithoutCompression()
+        [Theory]
+        [InlineData("8")]
```

Comment:
Looking online, the Theory+InlineData tags allow us to have the same test running with different parameters. However here, we're only using one InlineData tag. What's the difference compared to before?

Reply:
Ideally we want to test with more versions. Now we have attached it with a trait: https://github.com/microsoft/Oryx/pull/1294/commits/fe4b1aea7727001143caea98f28e3940991e7081#diff-096017d7f5e60abae085d1077d4a75861e2065edf62d853f9cfbf1385d66b706R142